BraintreeCore/src/main/java/com/braintreepayments/api/core/AnalyticsParamRepository.kt

```diff
@@ -26,10 +39,17 @@ class AnalyticsParamRepository(
         }
 
     /**
-     * Clears the session ID value from the repository
+     * Resets the [sessionId] and clears all other repository values.
+     *
+     * Note that this function is called in different spots of the SDK lifecycle for different payment modules. Some
+     * modules call reset during launch of the SDK. The PayPal module calls reset at the end of the payment flow to
+     * persist the [sessionId] value set from the Shopper Insights module.
```

Comment:
Can be handled in a separate PR but should we also call reset at the end of the payment flow for Venmo for shopper insights?

Reply:
Good catch! Yep, I'll create a follow up PR to add this to the Venmo module.